apps/drupal/src/Plugin/Preprocessor/README.md

```diff
@@ -0,0 +1,51 @@
+# Particle Preprocessor
+
+The Particle Preprocessor Plugin creates a system to preprocess theme templates.
+The system allows for processing via PHP rather than within Twig templates or
+overly long `.theme` files.
+
+## Preprocessors
+
+The Preprocessor directory contains the PHP work to preprocess templates.
+To match the Plugin system in Drupal core, these files should be named in the following manner:
+
+Entity - Bundle - View Mode 
+
+The Preprocessor directory is automatically scanned and loaded as a Class Map in ParticlePreprocessorManager.php.
+
+For now, each entity, bundle and view name should be separated by a capital letter.
+For example, if you have a `node--article--card-max`, name your php file: `NodeArticleCardmax.php`
```

Comment:
Anything you'd put into a theme preprocess function, e.g., `particle_preprocess_node__article()`, with the benefit that you can:
* Progressively build up those alterations all the way up the View mode level because inheritance
* Use xDebug inside these files.
* Organize them by their types (rather than have a ton of hooks)
* Easily export the pattern you've preprocessed in named chunks rather than relying on `content|without()` in twig

Reply:
@mikecrittenden Okay. I updated the README. Let me know if that makes it a bit more clear.